browseTemperature-weatherApiSimplified.iml

```diff
@@ -5,6 +5,8 @@
     <output-test url="file://$MODULE_DIR$/target/test-classes" />
```

Comment:
I'd remove *iml from the repo (please remember if you add to the repo iml files and after ignore them via .gitignore, it will not remove them ;) )

Reply:
removed